utilities/resources/envoy_filter.py

```
@@ -1,13 +1,13 @@
+from typing import Any
 # Generated using https://github.com/RedHatQE/openshift-python-wrapper/blob/main/scripts/resource/README.md
 
 from typing import Any
```

Flake8 (7.3.0)

[error] redefinition of unused 'Any' from line 1

(F811)